TheWarWithin/Priorities/EvokerDevastation.simc

```diff
@@ -48,33 +55,36 @@ actions+=/call_action_list,name=trinkets
 actions+=/run_action_list,name=aoe,strict=1,if=active_enemies>=3
 actions+=/run_action_list,name=st
 
-# AOE action list; Open with star before DR to save a global and start with a free EB
-actions.aoe+=/shattering_star,cycle_targets=1,if=cooldown.dragonrage.up&talent.arcane_vigor|talent.eternitys_span&active_enemies<=3
+# AOE action list; Open with star before DR to save a global and start with a free EB  BaumChange 2: Don't do as FS - Neutral
+actions.aoe+=/shattering_star,cycle_targets=1,if=(cooldown.dragonrage.up&talent.arcane_vigor|talent.eternitys_span&active_enemies<=3)&!talent.engulf
 actions.aoe+=/hover,use_off_gcd=1,if=raid_event.movement.in<6&!buff.hover.up&gcd.remains>=0.5&(buff.mass_disintegrate_stacks.up&talent.mass_disintegrate|active_enemies<=4)
 # Spend firestorm procs ASAP
 actions.aoe+=/firestorm,if=buff.snapfire.up&!talent.feed_the_flames
-actions.aoe+=/deep_breath,if=talent.maneuverability&talent.melt_armor&!cooldown.fire_breath.up&!cooldown.eternity_surge.up
-# Acquire the buff
-actions.aoe+=/firestorm,if=talent.feed_the_flames
+# BaumChange 1: Send DB before DR if FS FS FTF - +30k in 5T
+actions.aoe+=/deep_breath,if=talent.maneuverability&talent.melt_armor&!cooldown.fire_breath.up&!cooldown.eternity_surge.up|talent.feed_the_flames&talent.engulf&talent.imminent_destruction
+# Acquire the buff  BaumChange #3: Keep it from hardcasting firestorm while the SS debuff is running. I know the condition is garbage but it works - +20k in 5T
+actions.aoe+=/firestorm,if=talent.feed_the_flames&(!talent.engulf|cooldown.engulf.remains>4|cooldown.engulf.charges=0|(variable.next_dragonrage<=cooldown*1.2|!talent.dragonrage))
 # Grab Irid Red before Dragonrage without griefing extension
-actions.aoe+=/call_action_list,name=fb,if=talent.dragonrage&cooldown.dragonrage.up&(talent.iridescence|talent.scorching_embers)|empowering.fire_breath
+actions.aoe+=/call_action_list,name=fb,if=talent.dragonrage&cooldown.dragonrage.up&(talent.iridescence|talent.scorching_embers)&!talent.engulf|empowering.fire_breath
 actions.aoe+=/dragonrage
 # Tip ES at appropiate target count or when playing Flameshaper otherwise Tip FB
-actions.aoe+=/tip_the_scales,if=(!talent.dragonrage|buff.dragonrage.up)&(cooldown.fire_breath.remains<=cooldown.eternity_surge.remains&talent.engulf&talent.scorching_embers&cooldown.engulf.up|cooldown.eternity_surge.remains<cooldown.fire_breath.remains&(!talent.engulf|!cooldown.engulf.up|!talent.scorching_embers))
+actions.aoe+=/tip_the_scales,if=(!talent.dragonrage|buff.dragonrage.up)&(cooldown.fire_breath.remains<=cooldown.eternity_surge.remains|(cooldown.eternity_surge.remains<=cooldown.fire_breath.remains&talent.font_of_magic)&!talent.engulf)
+# BaumChange 2: Do as FS - Neutral
+actions.aoe+=/shattering_star,target_if=max:target.health.pct,if=(cooldown.dragonrage.up&talent.arcane_vigor|talent.eternitys_span&active_enemies<=3)&talent.engulf
```

Comment:
`target_if` is not supported.  In this case, `cycle_targets=1` should be okay in its place.  Check for any other `target_if` entries that might've slipped through.